apps/deploy-web/src/components/get-started/GetStartedStepper.tsx

```diff
@@ -88,41 +90,57 @@ export const GetStartedStepper: React.FunctionComponent = () => {
           onClick={() => (activeStep > 0 ? onStepClick(0) : null)}
           classes={{ label: cn("text-xl tracking-tight", { ["cursor-pointer hover:text-primary"]: activeStep > 0, ["!font-bold"]: activeStep === 0 }) }}
         >
-          Wallet
+          Billing
         </StepLabel>
 
         <StepContent>
-          <p className="text-muted-foreground">
-            You need at least {minDeposit.akt} AKT or {minDeposit.usdc} USDC in your wallet to deploy on Akash. If you don't have {minDeposit.akt} AKT or{" "}
-            {minDeposit.usdc} USDC, you can switch to the sandbox or ask help in our <ExternalLink href="https://discord.gg/akash" text="Discord" />.
-          </p>
+          {!isWalletConnected && (
+            <p className="text-muted-foreground">
+              In case you'd like to use a credit card for you billing, you may choose to start off with a trial and add you billing information later.
```

Comment:
```suggestion
              In case you'd like to use a credit card for you billing, you may choose to start off with a trial and add your billing information later.
```